distribution/lib/Standard/Base/0.0.0-dev/src/Data/Filter_Condition.enso

```diff
@@ -280,45 +280,45 @@ type Filter_Condition
     default_widget include_comparable:Boolean=True include_text:Boolean=True include_boolean:Boolean=True include_nullable:Boolean=True include_numeric:Boolean=True include_date:Boolean=True include_time:Boolean=True include_date_time:Boolean=True display:Display=..When_Modified -> Widget =
         options = Vector.build builder->
             equatable_types = make_any_selector add_text=include_text add_boolean=include_boolean add_number=include_numeric add_date=include_date add_time=include_time add_date_time=include_date_time
-            builder.append (Option "Equals" "..Equal" [["to", equatable_types]])
-            builder.append (Option "Not Equals" "..Not_Equal" [["to", equatable_types]])
+            builder.append (Option "..Equal" "..Equal" [["to", equatable_types]])
```

Comment:
If both `label` and `value` are now equal then I suggest to make use of the fact that `value` has default parameter set to `label`, so we could avoid setting it twice and duplicating the constant string twice risking typos.

We can do that by skipping providing the `value` parameter, letting it rely on a default and providing the next parameter by-name:
```suggestion
            builder.append (Option "..Equal" parameters=[["to", equatable_types]])
```

Reply:
What do you think? Alternatively we could create some helper function that constructs `Option` with both `value` and `label` having the same value, although IMHO the current constructor relying on the default should be good enough.

I think it would be good to avoid spelling these two string constants twice in so many cases - it is risking typos and other inconsistencies, whereas relying on them actually being equal guarantees that.

Reply:
Yes I like it. Have made that change.